data-prepper-core/build.gradle

```diff
@@ -25,6 +25,7 @@ dependencies {
     implementation "io.micrometer:micrometer-core:1.5.1"
     implementation "io.micrometer:micrometer-registry-prometheus:1.5.1"
     testImplementation "org.hamcrest:hamcrest:${versionMap.hamcrest}"
+    testImplementation "org.mockito:mockito-all:1.10.19"
```

Comment:
Might be time to wrangle all the mockito versions and stick them in the version map. 

[2 occurrences so far](https://github.com/opendistro-for-elasticsearch/Data-Prepper/search?q=org.mockito+filename%3A*.gradle), I'm guilty of adding one of them 😅

Reply:
Yep, I can do that!